Rakefile

```diff
@@ -44,6 +44,58 @@ namespace :version do
 
 end
 
+class OverrideReader
```

Comment:
chefdk is the only consumer for the moment. It's intended to be very local. I will make a `tasks/dependencies.rb` for all this.